README.md

```diff
@@ -10,7 +10,12 @@ The simplest case is to specify user name and password when starting
 the container:
 
 ```
-docker run -p 11021:21 -it --rm -e FTP_USER=scott -e FTP_PASS=tiger -e HOST=localhost mcreations/openwrt-ftp
+docker run -it --rm \
```

Comment:
Would you please leave the simplest case in the docs as it was before and *add* the possibility to specify the PASV ports?